src/index.ts

```diff
@@ -86,208 +144,262 @@ const allocationDatacap = async () => {
     });
 
     let issueInfoList: IssueInfo[] = [];
-    logGeneral(`issue n 0 Number of fetched comments: ${rawIssues.length}`);
-    for (const issue of rawIssues) {
-      try {
-        if (issue.labels.find((item: any) => item.name === "bot:readyToSign")) {
-          logGeneral(`Issue number ${issue.number} skipped --> bot:readyToSign is present`);
-          continue;
-        }
-        if (
-          issue.labels.find((item: any) => item.name === "status:needsDiligence")) {
-          logGeneral(`Issue number ${issue.number} skipped -->status:needsDiligence is present`);
-          continue;
-        }
-        if (issue.labels.find((item: any) => item.name === "status:Error")) {
-          logGeneral(`Issue number ${issue.number} skipped --> status:Error is present`);
-          continue;
-        }
+    let issueInfoListClosed = [];
+    logGeneral(`${LOG_PREFIX} 0 Number of fetched comments: ${rawIssues.length}`);
+    const promArr = []
+
 
-        //get all comments of a issue
-        const issueComments = await octokit.paginate(
-          octokit.rest.issues.listComments,
+    const commentsForEachIssue = await Promise.all(
+      rawIssues.map((issue: any) => new Promise<any>(async (resolve, reject) => {
```

Comment:
I think this promise is not longer necessary. You can just return the values from the map, and avoid the await in the paginate